### ISSUE TITLE
t/batches.t: UNIQUE constraint failed

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#1011]
When running the tests, `batches.t` returns "PASS", but something fails in the Perl code:
```
$ PERL_DL_NONLAZY=1 "/usr/local/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(1, 'inc', 'blib/lib', 'blib/arch')" t/batches.t
(...)
# Subtest: duplicate user should fail
DBD::SQLite::db do failed: UNIQUE constraint failed: users.username at /usr/home/matsd/.cpanm/work/1652791322.55779/Zonemaster-Backend-8.1.0/blib/lib/Zonemaster/Backend/DB.pm line 117.
    ok 1
    ok 2 - correct error type
    ok 3 - correct error message
    ok 4 - correct data type returned
    1..4
ok 7 - duplicate user should fail
(...)
All tests successful.
Files=1, Tests=8,  1 wallclock secs ( 0.03 usr  0.01 sys +  0.77 cusr  0.12 csys =  0.92 CPU)
Result: PASS
```

FreeBSD 13.0
SQLite 3.36.0 in p5-DBD-SQLite-1.70

**Comment by pnax** [comment]
This is the expected behavior of the test. This message is present for all databases. It is emitted by the driver and this is the expected behavior when you try to insert data that are already present (the UNIQUE constraint fails). The test properly catches the error.

**Comment by matsduf** [comment]
Is it possible to make the script out that is what should be expected? Not to confuse the users like me.

**Comment by matsduf** [comment]
Resolved by #1015
